bentoml/deployment/sagemaker/__init__.py

```diff
@@ -261,6 +259,57 @@ def init_sagemaker_project(sagemaker_project_dir, bento_path):
     return sagemaker_project_dir
 
 
+def _create_sagemaker_model(
+    sagemaker_client, bento_name, bento_version, ecr_image_path, api_name
+):
+    execution_role_arn = get_arn_role_from_current_aws_user()
+    model_name = create_sagemaker_model_name(bento_name, bento_version)
+
+    sagemaker_model_info = {
+        "ModelName": model_name,
+        "PrimaryContainer": {
+            "ContainerHostname": model_name,
+            "Image": ecr_image_path,
+            "Environment": {
+                "API_NAME": api_name,
+                "BENTO_SERVER_TIMEOUT": config().get('apiserver', 'default_timeout'),
+                "BENTO_SERVER_WORKERS": config().get(
+                    'apiserver', 'default_gunicorn_workers_count'
+                ),
+            },
+        },
+        "ExecutionRoleArn": execution_role_arn,
+    }
+
+    logger.info("Creating sagemaker model %s", model_name)
```

Comment:
can we make these two logging statement the same level? either both info or both debug?